numpy/lib/tests/test_arraypad.py

```diff
@@ -82,6 +125,30 @@ def test_check_maximum_2(self):
             )
         assert_array_equal(a, b)
 
+    def test_check_maximum_stat_length(self):
```

Comment:
@charris I feel obliged to point out that the recommended

```python
b = np.array([10]*25 + list(a) + [100]*20)
```
actually violates PEP8 by lacking spaces around the `*` operators. That said, I believe the grouping without spaces is clearer...